README.md

```diff
@@ -6,17 +6,17 @@ Ballerina WebSubHub Library
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/websubhub.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fwebsubhub)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-The WebSubHub library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+The WebSubHub package is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
 
-This library provides the capability to easily introduce publishers and hub that are WebSub-compliant. The ballerina/websubhub library comes with an in-built WebSub Hub service, which can be brought up by publishers that need to bring up a hub.
+This package provides the capability to easily introduce publishers and hub that are WebSub-compliant. The ballerina/websubhub package comes with an in-built WebSub Hub service, which can be brought up by publishers that need to bring up a hub.
```

Comment:
Is this statement correct? Shall we revisit the introduction in both `websub` and the `websubhub` modules?

Reply:
* Updated the module description and include api-doc link.